service/version_bundles.go

```diff
@@ -60,7 +60,7 @@ func NewVersionBundles() []versionbundle.Bundle {
 				},
 				{
 					Name:    "kubernetes",
-					Version: "1.8.1",
+					Version: "1.8.4",
```

Comment:
We should never modify established version bundles. We should rather deprecate this one and introduce another one. When we screwed existing clusters we have to fix them.

Reply:
this doesn't matter. we aren't using releases yet, 

and this is a WIP release, so then its ok to make changes to somethng that is WIP no?
So now its no longer in progress and the version is 1.8.4

Reply:
I too care about process, but lets be pragmatic here. What's the use of this deprecated 1.8.1 version lying around in the history.

Reply:
This matters. We are using version bundles for clusters already. 

In the future it should be ok to modify WIP version bundles but you modify a WIP version bundle and make it non-WIP. That does not sound right. 

It's use is to exist for all clusters that use it already. We cannot just change or drop it.